examples/convolutional_vae_util.py

```diff
@@ -21,6 +21,7 @@
 from prettytensor.pretty_tensor_class import Phase
 from prettytensor.pretty_tensor_class import PROVIDED
 
+
```

Comment:
This file is copy-and-pasted from [here](https://github.com/ikostrikov/TensorFlow-VAE-GAN-DRAW/blob/master/deconv.py). Since we modify it, maybe it makes sense to add to the top of this file something like:
```
# This file is taken from 
# https://github.com/ikostrikov/TensorFlow-VAE-GAN-DRAW/blob/master/deconv.py
# We only modify the formatting to pass PEP 8 standards.
#
# Copyright 2015 Google Inc. All Rights Reserved.
....
```